restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -170,16 +172,18 @@ protected void configure() {
 
     if (!enableSgv1) {
```

Comment:
NOTE: we will soon simply remove this conditional inclusion and drop registration of in-coordinator DocsV2, RESTv1. But README change can go in first; we are only keeping implementations to allow quick and safe rollback of the very first StagateV2 production deployment, if necessary.